guildedts.js.org / docs.guildedts.js.org

gamertike opened the pull request:
- [x] There is reasonable content on the page (see: [No Content](https://github.com/js-org/js.org/wiki/No-Content))
- [x] I have read and accepted the [Terms and Conditions](http://js.org/terms.html)

I have created a new website for my project using vercel, i have added `docs.guildedts` because the new website does not have docs implemented **yet** so i will remove `docs.guildedts` when it is. When i implement a guide into it i will also remove `guide.guildedts`

gamertike commented:
i require `// noCF`?? this prs checks failed

MattIPv4 commented:
Yes, you are requesting a sub-sub-domain, so `// noCF` is required as we cannot provide SSL/proxying for that.

gamertike commented:
@MattIPv4 fixed.

MattIPv4 reviewed (changes requested):
👋 Do you have a link to where we can preview the content for the new site?

gamertike commented:
> 👋 Do you have a link to where we can preview the content for the new site?

Yes, https://guildedts.vercel.app

indus commented:
I've changed and added the subdomains in JS.ORGs zonefile. But http://guildedts.github.io/guilded.ts has neither content nor a CNAME in place. Please fix this.
Please keep this PR open.

gamertike commented:
> 



> I've changed and added the subdomains in JS.ORGs zonefile. But http://guildedts.github.io/guilded.ts has neither content nor a CNAME in place. Please fix this or split this PR in two. Please keep this PR open.

Hi, i have added the `docs.guildedts.js.org` CNAME but that domain doesn't seem to be secure, can we change that? 
![image](https://user-images.githubusercontent.com/84046635/195877884-5e7b0e9f-ff57-4660-9175-ff3f740d0688.png)

gamertike commented:
> I've changed and added the subdomains in JS.ORGs zonefile. But http://guildedts.github.io/guilded.ts has neither content nor a CNAME in place. Please fix this or split this PR in two. Please keep this PR open.

The correct url is actually https://guildedts.github.io/docs, i have updated this pr to make that change.

indus commented:
Without the cf proxy you can still enforce https in the settings of your repo...Am 14.10.2022 16:56 schrieb Danielius Vaisnoras ***@***.***>:



I've changed and added the subdomains in JS.ORGs zonefile. But http://guildedts.github.io/guilded.ts has neither content nor a CNAME in place. Please fix this or split this PR in two. Please keep this PR open.

Hi, i have added the docs.guildedts.js.org CNAME but that domain doesn't seem to be secure, can we change that?


—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you commented.Message ID: ***@***.***>

gamertike commented:
> Without the cf proxy you can still enforce https in the settings of your repo...Am 14.10.2022 16:56 schrieb Danielius Vaisnoras ***@***.***>: I've changed and added the subdomains in JS.ORGs zonefile. But http://guildedts.github.io/guilded.ts has neither content nor a CNAME in place. Please fix this or split this PR in two. Please keep this PR open. Hi, i have added the docs.guildedts.js.org CNAME but that domain doesn't seem to be secure, can we change that? —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you commented.Message ID: ***@***.***>

Oh, it says that the TLS certificate is being provisioned, im assuming i just wait a while then try to enable https?

gamertike commented:
been like this for the past 5 hours 
![image](https://user-images.githubusercontent.com/84046635/195933403-e368d87a-c8aa-463c-a95e-77f7ac2bf9b0.png)

indus commented:
"Please allow 24 hours for this process to complete" ...
In case you need further support with the certificate you will have to talk to the GitHub support.
As both domains work this is ready to merge from JS.ORGs side. Or do you want me to wait?

gamertike commented:
> "Please allow 24 hours for this process to complete" ... In case you need further support with the certificate you will have to talk to the GitHub support. As both domains work this is ready to merge from JS.ORGs side. Or do you want me to wait?

I am happy enough for you to merge this pr.

indus commented:
Ok. Will do tomorrow...Am 14.10.2022 23:07 schrieb Danielius Vaisnoras ***@***.***>:

"Please allow 24 hours for this process to complete" ... In case you need further support with the certificate you will have to talk to the GitHub support. As both domains work this is ready to merge from JS.ORGs side. Or do you want me to wait?

I am happy enough for you to merge this pr.

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you commented.Message ID: ***@***.***>

gamertike commented:
@indus update: the process is still on the first step, not looking good .

indus commented:
Maybe you will have to host them together...
Nevertheless for now:
Welcome https://guildedts.js.org / http://docs.guildedts.js.org

gamertike commented:
> Maybe you will have to host them together... Nevertheless for now: Welcome https://guildedts.js.org / http://docs.guildedts.js.org

host them together?
